sxg_rs/src/header_integrity.rs

```diff
@@ -305,4 +304,46 @@ pub mod tests {
             EXPECTED_HEADER_INTEGRITY.as_bytes(),
         );
     }
+    #[async_std::test]
+    async fn out_of_order() {
+        use crate::utils::tests::{out_of_order, OutOfOrderState};
+        use futures::{
+            future::BoxFuture,
+            stream::{self, StreamExt},
+        };
+        struct OutOfOrderCache<F: Fn() -> BoxFuture<'static, Result<HttpResponse>>>(F);
+        #[async_trait(?Send)]
+        impl<F: Fn() -> BoxFuture<'static, Result<HttpResponse>>> HttpCache for OutOfOrderCache<F> {
+            async fn get(&self, url: &str) -> Result<HttpResponse> {
+                println!("get: url = {}", url);
```

Comment:
Should we remove this line and other `println`s?

Reply:
I think they're OK here. I left println statements in test only. The rustc test runner captures stdout so they're only displayed if the test fails. I'll go ahead and merge as-is, but feel free to remove in a follow-up if you disagree.